Makefile

```diff
@@ -107,7 +107,7 @@ container-build: module-check
 container-serve: module-check ## Boot the development server using container.
 	$(CONTAINER_RUN) --cap-drop=ALL --cap-add=AUDIT_WRITE --read-only \
 		--mount type=tmpfs,destination=/tmp,tmpfs-mode=01777 -p 1313:1313 $(CONTAINER_IMAGE) \
-		hugo server --buildFuture --environment development --bind 0.0.0.0 --destination /tmp/public --cleanDestinationDir --noBuildLock
+		hugo serve --minify --renderToMemory --buildFuture --environment development --bind 0.0.0.0 --destination /tmp/public --cleanDestinationDir --noBuildLock
```

Comment:
Does this mean that you needed to run `npm ci` outside the container first? If so:
- let's document that
- let's get a mini security review

Reply:
I'd really prefer to keep the setup where `npm ci` runs within the container image build step. I hope we can.

:thought_balloon: to make that work right, we should really change `scripts/hash-files.sh` to include `package-lock.json`

Reply:
I've reverted this change since it isn't essential to the 0.5.1 upgrade.

All good comments. Could you create a separate issue to track those, since they're outside of the scope of this PR. Thanks!